### ISSUE TITLE
Configure aws git access

### ISSUE THREAD

**Issue by reubano** [openpeoria/AWS-CKAN#7]
- [x] [codecommit](https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up.html)
- [x] [codedeploy](https://docs.aws.amazon.com/codedeploy/latest/userguide/getting-started-codedeploy.html)
- [ ] [codepipeline](https://docs.aws.amazon.com/codepipeline/latest/userguide/tutorials-simple-codecommit.html)

**Comment by reubano** [comment]
Got stuck at codepipeline step 10.

The pipeline starts running after it is created. It downloads the code from your CodeCommit repository and creates a CodeDeploy deployment to your EC2 instance. You can view progress and success and failure messages as the CodePipeline sample deploys the webpage to the Amazon EC2 instance in the CodeDeploy deployment.

https://console.aws.amazon.com/codesuite/codepipeline/pipelines?region=us-east-1

![image](https://user-images.githubusercontent.com/157864/78729562-f889de80-78ff-11ea-96be-40884370e18e.png)

**Comment by reubano** [comment]
I thought `codecommit` would be easier to integrate than `github`, but that [may not be the case](https://medium.com/@sithum/automate-static-website-deployment-from-github-to-s3-using-aws-codepipeline-16acca25ebc1)...

**Comment by sottom** [comment]
This sounds pretty good to me. CKAN isn't a static site though, right?

**Comment by reubano** [comment]
Correct. Thats just showing the github part. For us we wont use the s3 stuff. Austin used it for his site, so I asked which tute he found.

**Comment by sottom** [comment]
Ideally this would be done with github.

**Comment by sottom** [comment]
I tried to deploy from github to EC2 instance and got stuck at CodePipeline, same as Reuban.
This was the error:
![image](https://user-images.githubusercontent.com/18564176/87953241-06571000-ca68-11ea-9392-f68719082d4d.png)


## Steps to Reproduce
- Followed all instructions in [this tutorial](https://seanjziegler.com/deploying-code-from-github-to-aws-ec2-with-codepipeline/)
  - I copied the `/lib/ckan/default/src/ckan` directory from the EC2 instance to my local machine and created a git repository in my own Github account.
  - I tried to push code to the existing CKAN EC2 instance that is running for OpenPeoria.
  - I used an existing CodeDeploy Role that Reuben made on his try.

## Next Steps
- [ ] Try deploying random code to a new EC2 instance to see if the process is failing because of the codebase or existing ec2 instance.

**Comment by reubano** [comment]
Thanks! If you haven't done so already, can you document the symlink we made during our 1-on-1 session?

**Comment by sottom** [comment]
I updated my comment above to reflect the symlink, and I made a PR [here](https://github.com/nerevu/handbook/pull/15) with the same information. Once the PR is merged, the link in my comment above will link to the correct section in the handbook.

**Comment by reubano** [comment]
https://aws.amazon.com/codestar/features/
